### PR TITLE
Update ICameraHandler.cs

### PR DESCRIPTION
**Pull request by dfrankes**
PR for issue #155 

The playerLocation will only be send if connected to the server (playerCount > 0), also added a check to make sure the camera object exist.

also changed the hook method to LateUpdate, this should help a little bit with peformance.

### CODE REVIEW

**Review by DominicMaas — APPROVED** [review]

**Comment by DominicMaas** [comment]
Code looks good, I just need to physically test it.

File: src/Injections/ICameraHandler.cs [hunk 1]
```diff
@@ -4,40 +4,50 @@
...
-                    PlayerColor = JoinGamePanel.playerColor
-                });
+                    // Store camera rotation and position
+                    playerCameraPosition_last = _position;
+                    playerCameraRotation_last = _rotation;
+
+                    // Send info to all clients
+                    Command.SendToAll(new PlayerLocationCommand
+                    {
+                        PlayerName = MultiplayerManager.Instance.CurrentClient.Config.Username,
```
Comment (kaenganxt) [inline src/Injections/ICameraHandler.cs]: This causes an exception on the server (where the client Config is not set). You need to get the name differently when the game acts as the server (`MultiplayerManager.Instance.CurrentRole == MultiplayerRole.Server`) by using `MultiplayerManager.Instance.CurrentServer.Config.Username`.

File: src/Injections/ICameraHandler.cs [hunk 2]
```diff
@@ -4,40 +4,50 @@
...
-            Transform transform = ___m_camera.transform;
-            Quaternion _rotation = transform.rotation;
-            Vector3 _position = transform.position;
-
-            // Check if the camera moved
-            if (Vector3.Distance(_position, playerCameraPosition_last) > 1 || playerCameraRotation_last != _rotation)
+            // Make sure the camera object is defined, this should always happen but for some strange reason it didnt (look bellow)
+            // This is not happening if i test it localy. (debug using AllocConsole in CSM.cs)
+            // https://github.com/DominicMaas/Tango/issues/155
+            if (___m_camera && MultiplayerManager.Instance.PlayerList.Count > 0)
```
Comment (kaenganxt) [inline src/Injections/ICameraHandler.cs]: Please use `MultiplayerManager.Instance.CurrentRole != MultiplayerRole.None` instead of the player list check.

**Review by kaenganxt — CHANGES_REQUESTED** [review]
Thank you for your contribution!
The issue with spamming errors doesn't seem to be fixed, I believe the exception is thrown by always accessing the current client (which is not set on the server) (see specific comment)
The command is also still sent very often, I would prefer if it was sent only every x (e.g. 1 or 2) seconds (when changed).

File: src/Injections/ICameraHandler.cs (hunk 2 above)
Comment (dfrankes) [reply src/Injections/ICameraHandler.cs]: I have applied the changes you have requested

File: src/Injections/ICameraHandler.cs (hunk 1 above)
Comment (dfrankes) [reply src/Injections/ICameraHandler.cs]: Fixed in latest commit

**Review by kaenganxt — APPROVED** [review]
Thanks for your changes! I just added some changes like the fix for #164.

**Pull request merged**